neard/build.rs

```diff
@@ -45,16 +76,28 @@ fn get_git_version() -> Result<String> {
         let path = git_dir.join(subpath).canonicalize()?;
         println!("cargo:rerun-if-changed={}", path.display());
     }
-    let out = command("git", &["describe", "--always", "--dirty=-modified"]);
-    match out.as_ref().map(|ver| String::from_utf8_lossy(&ver)) {
-        Ok(std::borrow::Cow::Borrowed(version)) => Ok(version.trim().to_string()),
-        Ok(std::borrow::Cow::Owned(version)) => Err(anyhow!("git: invalid output: {}", version)),
-        Err(msg) => {
+    command(std::ffi::OsStr::new("git"), &["describe", "--always", "--dirty=-modified"]).or_else(
+        |error| {
             println!("cargo:warning=unable to determine git version");
-            println!("cargo:warning={}", msg);
+            println!("cargo:warning={}", error);
             Ok("unknown".to_string())
-        }
-    }
+        },
+    )
+}
+
+/// Returns rustc version.
+fn get_rustc_version() -> Result<String> {
+    command(&env_or("RUSTC", "rustc"), &["-vV"])
```

Comment:
Why do we want to run command by hand, if we already depend on `rustc_version` and can use [rustc_version](https://docs.rs/rustc_version/latest/rustc_version/index.html)::[version_meta](https://docs.rs/rustc_version/latest/rustc_version/fn.version_meta.html#)?